discord/utils.py

```diff
@@ -340,12 +375,11 @@ def deprecated(
     since: str | None = None,
     removed: str | None = None,
     reference: str | None = None,
-    stacklevel: int = 3,
+    stacklevel: int = 1,
```

Comment:
Why

Reply:
Typing's thing for some reason changes the levels in some way and to keep a correct stacktrace I had to change it. Admittedly I just tried and see.